simple-service-webapp-tests/src/test/java/com/example/TestTest.java

```diff
@@ -105,31 +106,53 @@ public void testPutMethod() throws IOException {
     }
 
     @Test
-    public void getLoginToken() throws IOException {
+    public void getLoginToken() throws IOException,ParseException {
         String token = authenticate();
 
         System.out.println(token);
     }
 
 
-    public String authenticate() throws IOException {
+    public String authenticate() throws IOException, ParseException {
         HttpTransport transport = new NetHttpTransport();
         HttpRequestFactory requestFactory = transport.createRequestFactory();
+        //should read from file
 
-        GenericUrl url = new GenericUrl("http://localhost:8080/api/authentication");
+        Properties prop = new Properties();
+        InputStream input = null;
 
-        Credentials credentials = new Credentials("abcde", "12345");
-
-        HttpRequest request =
-                requestFactory
-                        .buildPostRequest(url, new ByteArrayContent("application/json", mapper.writeValueAsString(credentials).getBytes()));
+        try {
 
+            input = new FileInputStream("C:\\simple-service-webapp\\simple-service-webapp-tests\\src\\test\\resources\\credentials.properties");
```

Comment:
It would be nice that you have a util class that reads the properties, and strong typed methos for getting the values.